js/draw.js

```diff
@@ -0,0 +1,16 @@
+import {photos} from './mock.js';
+
+const photosElement = document.querySelector('.pictures');
+const photoTemplate = document.querySelector('#picture').content.querySelector('.picture');
+const fragment = document.createDocumentFragment();
+
+photos.forEach(({url, description, likes, comments})=>{
+	const photoElement = photoTemplate.cloneNode(true);
+	photoElement.querySelector('.picture__img').src = url;
+	photoElement.querySelector('.picture__img').alt = description;
```

Comment:
Можно искать эту штуку один раз
```suggestion
	const imageElement = photoElement.querySelector('.picture__img');
	imageElement.src = url;
	imageElement.alt = description;
```